cognite/neat/_rules/analysis/_base.py

```diff
@@ -524,6 +549,63 @@ def class_uri(self, class_: ClassEntity) -> URIRef | None:
         else:
             return None
 
+    def property_uri(self, property_: InformationProperty) -> URIRef | None:
+        if (instance_source := property_.instance_source) and isinstance(instance_source.traversal, SingleProperty):
+            prefix = instance_source.traversal.property.prefix
+            suffix = instance_source.traversal.property.suffix
+
+            if namespace := self.information.prefixes.get(prefix):
+                return namespace[suffix]
+        return None
+
     @property
     def multi_value_properties(self) -> list[InformationProperty]:
         return [prop_ for prop_ in self.information.properties if isinstance(prop_.value_type, MultiValueTypeInfo)]
+
+    @property
+    def query_config_by_view_id(
+        self,
+    ) -> dict[dm.ViewId, dict[str, None | URIRef | dict[URIRef, str]]]:
+        _ = self.information
+        _ = self.dms
+
+        # caching results for faster access
+        classes_by_neat_id = self.class_by_neat_id
+        properties_by_class = self.properties_by_class(include_ancestors=True)
+        logical_uri_by_view = self.logical_uri_by_view
+        logical_uri_by_property_by_view = self.logical_uri_by_property_by_view(include_ancestors=True)
+        information_properties_by_neat_id = self.properties_by_neat_id()
+
+        config: dict[dm.ViewId, dict[str, None | URIRef | dict[URIRef, str]]] = {}
+
+        for view in self.dms.views:
+            # this entire block of sequential if statements checks:
+            # 1. connection of dms to info rules
+            # 2. correct paring of information and dms rules
+            # 3. connection of info rules to instances
+            if (
+                (neat_id := logical_uri_by_view.get(view.view))
+                and (class_ := classes_by_neat_id.get(neat_id))
+                and (uri := self.class_uri(class_.class_))
```

Comment:
This method, `self.class_uri`, might be worth a discussion. Should we introduce an `Instance Source` for classes in the Information Rules, which sets the RDF.type explicitly instead of inferring it like we do here? It seems risky.